src/tools/miropt-test-tools/Cargo.toml

```diff
@@ -5,3 +5,4 @@ edition = "2021"
 
 [dependencies]
 regex = "1.0"
+lazy_static = "1"
```

Comment:
This should be avoidable by using https://doc.rust-lang.org/stable/std/sync/struct.OnceLock.html.